ipa-core/src/protocol/dp/mod.rs

```diff
@@ -21,6 +21,36 @@ use crate::{
         BitDecomposed, FieldSimd, TransposeFrom, Vectorizable,
     },
 };
+
+/// Struct to hold noise parameters, contains internal values not received from the client
```

Comment:
Can you please document these parameters and also give an explanation for default values as well?